### PR TITLE
do not respect crd status generation.

### PR DESCRIPTION
**Pull request by n3wscott**
## Proposed Changes

  * allow k8s >1.11 to apply source yaml.

**Release Note**
```release-note
Fix for applying source yaml to k8s > 1.11
```

### CODE REVIEW

File: config/crds/sources_v1alpha1_containersource.yaml [hunk 1]
```diff
@@ -38,6 +38,8 @@ spec:
             image:
               minLength: 1
               type: string
+            serviceAccountName:
+              type: string
```
Comment (bbrowning) [inline config/crds/sources_v1alpha1_containersource.yaml]: These serviceAccountName changes look unrelated to this specific fix - probably slipped in when generating the manifests based on some local changes to the container and kubernetes event source types?

**Comment by grantr** [comment]
This was fixed upstream in https://github.com/kubernetes-sigs/controller-tools/pull/80. We're using a forked version of that dep to fix a different issue, so @n3wscott will try to get that patch into our fork. Hopefully that will make  this change unnecessary.

File: config/crds/sources_v1alpha1_containersource.yaml (hunk 1 above)
Comment (n3wscott) [reply config/crds/sources_v1alpha1_containersource.yaml]: someone forgot to run the gen command.

Comment (n3wscott) [reply config/crds/sources_v1alpha1_containersource.yaml]: @grantr ^^

but I am not sure why the verify script did not catch this.

Comment (grantr) [reply config/crds/sources_v1alpha1_containersource.yaml]: :hand: This should have gone into #60, but I'm also curious why the verify script didn't work.

**Comment by bbrowning** [comment]
Thanks for the quick fix!
